constants/index.js

```diff
@@ -11,5 +11,15 @@ module.exports = {
         prolonged general anaesthesia.`,
     D: `Use of neromuscular blocking agents`,
     F: 'Other'
+  },
+  pilEndorsmentStatuses: {
+    'ntco-endorsed': {
+      label: 'Yes',
+      hint: `I confirm that the applicant holds the neccessary training or experience to carry out the categories of procedures listed in this applciation`
```

Comment:
We probably shouldn't use backticks for content which isn't template literals.

Reply:
Same typo here too: `application`

Reply:
Actually, I'm not sure why this content is here at all. Given it's content, and is also defined in the content files. Remove it.